inc/setup/cachify.memcached.nginx.php

```diff
@@ -71,7 +71,7 @@
 		<?php
 		printf(
 			/* translators: code */
-			esc_html__( 'If you have errors please try to change %1$s to %2$s This forces IPv4 because some servers that allow IPv4 and IPv6 are configured to bind memcached to IPv4 only.', 'cachify' ),
+			esc_html__( 'If you have errors please try to change %1$s to %2$s This forces IPv4 because some servers that allow IPv4 and IPv6 are configured to bind Memcached to IPv4 only.', 'cachify' ),
```

Comment:
I believe (judging from what I read on the webpage) the name is `memcached`. It is written as `Memcached` only in headlines.

Reply:
Done